src/pages/Question/CommentItem/CommentItem.module.scss

```
@@ -7,23 +7,34 @@
   border-bottom-style: solid;
   border-bottom-width: 1px;
 
+  .commetEdit {
+    width: 800px;
+
+    .container {
+      width: 700px;
+    }
+  }
+
   form {
     display: flex;
     margin-top: 3px;
-
-    textarea {
-      width: 500px;
-      margin: 0 10px 3px 0;
-      font-family: sans-serif;
-    }
+    width: 100%;
 
     button {
       height: 30px;
+      margin-left: 12px;
     }
   }
 
-  span {
+  p {
+    word-wrap: break-word;
+    margin: 10px;
+    flex-wrap: wrap;
+  }
```

줄바뀜 문제는 `display: inline` 넣어주면 대부분 해결됩니다 :)
```suggestion
  p {
    word-wrap: break-word;
    margin: 10px;
    flex-wrap: wrap;
    display: inline;
  }
```

---

감사합니다!ㅎㅎ